index.html

```diff
@@ -2066,72 +2062,21 @@ <h3>Securing Mechanisms</h3>
 
         <p>
 Methods of securing <a>verifiable credentials</a> or
-<a>verifiable presentations</a> that embed a proof in the data model MUST use
-the <code>proof</code> <a>property</a>.
+<a>verifiable presentations</a> that embed a proof in the data model MAY use a
+the <code>proof</code> <a>property</a>,
+when specified by the securing mechanism.
         </p>
```

Comment:
I am afraid the same issue arises as in my comment on https://github.com/w3c/vc-data-model/pull/1389#issuecomment-1860849474. 

_If_ any securing mechanism _MAY_ use a `proof` property, then this property must be formally defined in the VCDM specification. At the moment, because it is formally defined in the DI spec, using `proof` means that a third party securing mechanism, which is ***not*** DI, would have a dependency on the DI spec. That is a different layering violation that we would introduce.

If the `proof` property is formally _defined_ in the VCDM as _one possible way of hooking a securing mechanism into VCDM_ then I believe we are fine (provided the DI spec is also updated accordingly).

I give more LD terms in https://github.com/w3c/vc-data-model/pull/1389#issuecomment-1860849474, but I believe the underlying problem is identical.